java-example/README.MD

```diff
@@ -15,14 +15,9 @@ This guide shows you how to auto instrument your Java application using the Open
 Follow these steps to install the Wavefront proxy using Docker. See [Install a Proxy](https://docs.wavefront.com/proxies_installing.html#install-a-proxy) to find other options for installing the proxy in your environment.
 
 1. Make sure you have **Proxies** permission.
-    
-    ![A screenshot showing the proxies permission selected.](images/java_auto_instrumentation_proxies permission.png)
```

Comment:
I'm no markdown expert, but I noticed that the file name: ```java_auto_instrumentation_proxies permission.png``` contains a space. Could this be why the image isn't showing up?  I'm am wondering if you have tried renaming the image in the images directory to read ```java_auto_instrumentation_proxies_permission.png``` and changing the above link to match.